masonite/drivers/UploadS3Driver.py

```diff
@@ -17,6 +20,9 @@ def store(self, fileitem, location=None):
         s3 = session.resource('s3')
 
         s3.meta.client.upload_file(
-            file_location, self.config.DRIVERS['s3']['bucket'], fileitem.filename)
+            file_location,
+            self.config.DRIVERS['s3']['bucket'], fileitem.filename
```

Comment:
Let's put that third parameter on the next line as well. So just move fileitem.filename down the the next line too.